ksuid.go

```diff
@@ -179,7 +183,11 @@ func Parse(s string) (KSUID, error) {
 	dst := [byteLength]byte{}
 
 	copy(src[:], s[:])
-	fastDecodeBase62(dst[:], src[:])
+
+	if err := fastDecodeBase62(dst[:], src[:]); err != nil {
+		return Nil, errStrValue
```

Comment:
This seems slightly leaky. Would `fastDecodeBase62` never error for a reason other than `errStrValue`? The current implementation wouldn't, but is that ok to assume for the future?

Reply:
I think it's fine, there's little chance for this code to change, plus it's an implementation detail so we're free to change it if needed be.